packages/core/src/core.jsx

```diff
@@ -117,7 +117,7 @@ export default (
       render() {
         const { props } = this.state
         if (isInError(this.props, this.state, this.context)) {
-          return <ErrorIndicator {...props} />
+          return ErrorIndicator === undefined ? null : <ErrorIndicator {...props} />
```

Comment:
Using the code below doesn't work?
```jsx
ErrorIndicator && <ErrorIndicator {...props} />
```

Reply:
Yeah... I don't know what happens 😱

Reply:
Ho I see right know why I did this:
 - react render nothing if you return `null` but not `undefined`